packages/gafl-webapp-service/src/routes/journey-definition.js

```diff
@@ -75,11 +80,17 @@ export default [
       [CommonResults.OK]: {
         page: DATE_OF_BIRTH
       },
+      [CommonResults.AMEND]: {
+        page: CHANGE_LICENCE_OPTIONS
+      },
       [CommonResults.SUMMARY]: {
         page: LICENCE_SUMMARY
       }
     },
-    backLink: s => (s.fromSummary ? LICENCE_SUMMARY.uri : LICENCE_FOR.uri)
+    backLink: async status => {
+      const backLinkResult = await backLinkHandler(status, LICENCE_FOR.uri)
+      return backLinkResult
+    }
```

Comment:
Please rewrite to be `backLink: status => backLinkHandler(status, LICENCE_FOR.uri)`